cli/cmd/search.go

```diff
@@ -37,6 +44,11 @@ func internalSearchModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	if len(args) != 1 {
 		return fmt.Errorf("Incorrect arguments")
 	}
-	err := search.SearchVersions(cmdCtx, args[0])
+	if local {
+		err = search.SearchVersionsLocal(cmdCtx, args[0])
```

Comment:
Add integration and unit tests for local versions searching.

Reply:
added integration tests